packages/cli/README.md

```diff
@@ -21,7 +21,7 @@ npm install -g @celo/celocli@beta
 ```
 
 > [!TIP]
-> IF you're on macOS or Linux, you may use [Homebrew](https://brew.sh/) to install this. `brew install celocli`
+> IF you're on macOS or Linux, you may use [Homebrew](https://brew.sh/) to install this. `brew install celo-org/brew/celocli`
```

Comment:
is that right? I got an error

but 

```bash
brew tap celo-org/brew
brew install celocli
```

works

Reply:
```bash
brew celo-org/brew/celocli
Error: Unknown command: brew celo-org/brew/celocli
```

Reply:
oops should be `brew install celo-org/brew/celocli` but i think i like yours better for clarity